library/src/scripts/bootstrap.ts

```diff
@@ -20,6 +21,9 @@ import { bootstrapLocales } from "@library/locales/localeBootstrap";
 const debugValue = getMeta("context.debug", getMeta("debug", false));
 debug(debugValue);
 
+const translationDebugValue = getMeta("context.translationDebug", getMeta("translationDebug", false));
```

Comment:
What's the reason for the nesting fallback? This value should always be in the same place.

Reply:
removed second getMeta call